rewrite-core/src/test/java/org/openrewrite/marker/GitProvenanceTest.java

```diff
@@ -83,9 +91,9 @@ void getOrganizationNameWithBaseUrl(String gitOrigin, String baseUrl, String org
 
     @ParameterizedTest
     @MethodSource("remotes")
-    void getRepositoryName(String remote) {
+    void getRepositoryName(String remote, String org, String repo) {
```

Comment:
Can we collapse this and `getOrganizationName` above to get rid of the unused argument warnings?

Reply:
Done. Thanks for the suggestion!